src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleLines = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `styleLines` as it does not convey the content of the variable clearly. Consider a more descriptive name that reflects the nature of the data, such as `styleDeclarations` or `cssRules`.